docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+web:
+    build: .
+    volumes:
+        - .:/project
+    ports:
+        - "8000:8000"
+    links:
+        - db
+    command: make install
```

Comment:
Maybe I don't understand what the install target does, but is that really what you want to run?